ember-client/shared/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -374,17 +379,17 @@ object EmberClientBuilder extends EmberClientBuilderCompanionPlatform {
     val acgFixedThreadPoolSize: Int = 100
     val chunkSize: Int = 32 * 1024
     val maxResponseHeaderSize: Int = 4096
-    val idleConnectionTime = org.http4s.client.defaults.RequestTimeout
+    val idleConnectionTime: FiniteDuration = org.http4s.client.defaults.RequestTimeout
     val timeout: Duration = org.http4s.client.defaults.RequestTimeout
 
     // Pool Settings
-    val maxPerKey = { (_: RequestKey) =>
+    val maxPerKey: RequestKey => Int = { (_: RequestKey) =>
       100
     }
-    val maxTotal = 100
-    val idleTimeInPool = 30.seconds // 30 Seconds in Nanos
-    val additionalSocketOptions = List.empty[SocketOption]
-    val userAgent = Some(
+    val maxTotal: Int = 100
+    val idleTimeInPool: FiniteDuration = 30.seconds // 30 Seconds in Nanos
+    val additionalSocketOptions: List[SocketOption] = List.empty[SocketOption]
+    val userAgent: Option[`User-Agent`] = Some(
```

Comment:
MiMa is complaining about this one, but only on Scala 2 I think (it wants `Some`). So idk if we can put a type annotation that works across all Scala versions.